scripts/scripts.js

```diff
@@ -1,6 +1,5 @@
 import {
   sampleRUM,
-  /* buildBlock, */
```

Comment:
why did you delete that? I commented out temporarily but we'll use it soon enough and I didn't want to forget to add it here :-)

Reply:
@helms-charity 
I added it back, I uncommented it first,  Lint wasn't passed, I tried to comment it out but deleted it by mistake.
Sorry about it.